source/JsMaterialX/JsMaterialXCore/JsElement.cpp

```diff
@@ -22,10 +22,6 @@ namespace mx = MaterialX;
     BIND_MEMBER_FUNC("setValue" #NAME, mx::ValueElement, setValue<T>, 1, 2, const T&, stRef)
 
 #define BIND_ELEMENT_CHILD_FUNC_INSTANCE(NAME, T)                                                 \
-    BIND_MEMBER_FUNC("addChild" #NAME, mx::Element, addChild<T>, 0, 1, stRef)                     \
```

Comment:
@ashwinbhat I see that the one build failure is happening in our JavaScript test suite, and I wonder if it would address the issue if we restored these lines in `JsElement.cpp` while we're working out the details of the C++ API.

Reply:
Maybe just comment it out ? I could try to build locally to help resolve wrapper issues.

Reply:
@kwokcb I'll restore the current state